eZ/Publish/Core/Persistence/Legacy/User/Gateway.php

```diff
@@ -83,4 +83,11 @@
      * @param mixed $roleId
      */
     abstract public function removeRole($contentId, $roleId);
+
+    /**
+     * Remove role from user or user group, by assignment ID.
+     *
+     * @param mixed $assignmentId
+     */
+    abstract public function removeRoleByAssignmentId($assignmentId);
```

Comment:
`removeRoleAssignmentById()` ? Seems to clarify what it's supposed to, especially when reading code from gateway below :smiley: